kerl

```diff
@@ -2099,6 +2099,38 @@ get_active_path() {
     fi
 }
 
+fix_otp_installations() {
+    if [ -f "$KERL_BASE_DIR"/otp_installations ]; then
+        nonexisting_names=""
```

Comment:
i would use "missing" - that's the more natural word for this situation :woman_shrugging:

Reply:
Cool. Naming, right? I'll change it.

Reply:
f9c546a